blueprints/tr-load-tests.md

```diff
@@ -102,7 +109,7 @@ How will existing tables and columns be changed?
 What are the column data types and modifiers?
 What are the FK references and constraints?
 -->
-No Data Model impact is anticipated.
+A database migration should be added to seed a new Operations-level user for use with the TR Ultimate Test Harness.
```

Comment:
Do all install environments really need to support running the test harness? Surely we only want to run it against testing environments? What new Permission would this user need that existing users don't have?

Reply:
As users cannot be deleted through the API and any valid installation of ATC follows the setup process that runs `postinstall` which will create a default user, no supported environments have no users, and as of #6324 a Role will always exist that has all Permissions. So why do all new installs/upgrades of ATC need to create a new user just for running tests?

Reply:
Good points, removed mention of the *user creation* migration section in f202723c5c.